astropy/io/ascii/tests/test_cds_header_from_readme.py

```diff
@@ -21,6 +21,19 @@ def read_table3(readme, data):
     return ascii.read(data, readme=readme)
 
 
+def test_description():
+    readme = 't/cds/description/ReadMe'
+    data = 't/cds/description/table.dat'
+    for read_table in (read_table1, read_table2, read_table3):
+        table = read_table(readme, data)
+        assert_equal(len(table), 10)
+        assert_equal(table['Cluster'].description, 'Cluster name')
+        assert_equal(table['Star'].description, '')
+        assert_equal(table['Wave'].description, 'wave? Wavelength in Angstroms')
+        assert_equal(table['El'].description, 'a')
+        assert_equal(table['ion'].description, '- Ionization stage (1 for neutral element)')
+
```

Comment:
That's the point of the test, to confirm that the missing value markup does not end up in the description.